docs/templates/plugin.rst.j2

```diff
@@ -142,7 +142,7 @@ Parameters
                                 {% elif choice is sameas false %}
                                     {% set choice = 'no' %}
                                 {% endif %}
-                                {% if (value.default is string and value.default == choice) or (value.default is iterable and value.default is not string and choice in value.default) %}
+                                {% if (value.default is string and value.default|lower == choice) or (value.default is iterable and value.default is not string and choice in value.default) %}
```

Comment:
It looks like this affects non-boolean values, while the intention is to fix booleans, right?

Reply:
It converts all boolean value to string 'yes' 'no' in some lines above. So here boolean is a string, I can't distinguish them anymore.

Reply:
yea, but you should convert everything to `yes` lowercase, right?

Reply:
everything you mean the choice here should be converted too?

Reply:
yes, i'm talking about choice.